theano/tensor/nnet/tests/test_abstract_conv.py

```diff
@@ -1548,3 +1549,35 @@ def test_compare_1D_and_2D_upsampling_values(self):
         f_1D = theano.function([], mat_1D, mode=self.compile_mode)
         f_2D = theano.function([], mat_2D, mode=self.compile_mode)
         utt.assert_allclose(f_1D(), f_2D(), rtol=1e-06)
+
+
+class TestConv2dTranspose(unittest.TestCase):
+    mode = None
+
+    def test_interface(self):
+        """Test conv2d_transpose wrapper.
+
+        This method tests that the order of the filter's
+        axes expected by the function produces the correct
+        output shape.
+
+        """
+        output = theano.function(
+            inputs=[],
+            outputs=conv2d_transpose(input=tensor.ones((2, 2, 4, 4)),
+                                     filters=tensor.ones((2, 1, 4, 4)),
+                                     output_shape=(2, 1, 10, 10),
+                                     input_dilation=(2, 2)),
+            mode=self.mode)()
+        expected_output = numpy.array(
+            [[[[2, 2, 4, 4, 4, 4, 4, 4, 2, 2],
+               [2, 2, 4, 4, 4, 4, 4, 4, 2, 2],
+               [4, 4, 8, 8, 8, 8, 8, 8, 4, 4],
+               [4, 4, 8, 8, 8, 8, 8, 8, 4, 4],
+               [4, 4, 8, 8, 8, 8, 8, 8, 4, 4],
+               [4, 4, 8, 8, 8, 8, 8, 8, 4, 4],
+               [4, 4, 8, 8, 8, 8, 8, 8, 4, 4],
+               [4, 4, 8, 8, 8, 8, 8, 8, 4, 4],
+               [2, 2, 4, 4, 4, 4, 4, 4, 2, 2],
+               [2, 2, 4, 4, 4, 4, 4, 4, 2, 2]]]] * 2)
```

Comment:
Given that this is a very thin wrapper around `conv2d_grad_wrt_inputs`, I think it's sufficient to make sure that there aren't any surprises in the interface, e.g. with regards to the order of input/output channel axes.